src/main/kotlin/net/onelitefeather/clipboardconnect/services/SyncService.kt

```diff
@@ -54,7 +50,8 @@ class SyncService @Inject constructor(private val config: FileConfiguration, pri
     private val redisson: RedissonClient = buildRedis()
     private val topicName = "ClipboardConnect"
     private val serverName = config.getString("servername") ?: "Unknown"
```

Comment:
So default is unknown? Is there a documentation / readme so that users know they have to set it up for them or is this in the ingame setup?

Reply:
Thats a fallback before the plugin will be throw some errors